File: source_code/CosmeticStoreAPI/CosmeticStoreAPI/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CosmeticStoreAPI.Models
{
    public class Order
    {
        public string id { get; set; }
        public string user_id { get; set; }
        public string shoppingcart_id { get; set; }
        public DateTime date { get; set; }
        public string place { get; set; }
        public string city_id { get; set; }
        public string delivery_id { get; set; }
        public string payment_id { get; set; }
        public bool status { get; set; }

    }
}

File: source_code/CosmeticFrontEnd/app/models/Order.js
var Order = function (id, user_id, date, place, city_id, delivery_id, payment_id, status, shoppingcart_id) {
    this.id = id;
    this.user_id = user_id;
    this.date = date;
    this.place = place;
    this.city_id = city_id;
    this.delivery_id = delivery_id;
    this.payment_id = payment_id;
    this.status = status;
    this.shoppingcart_id = shoppingcart_id;
}